stories/layout.jsx

```diff
@@ -0,0 +1,108 @@
+// @flow
+
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { withKnobs, radios } from '@storybook/addon-knobs';
+
+import ProductPageHero, { HeroHanger, HeroWrapper } from 'components/productPage/productPageHero/productPageHero';
+import Content, { Divider, Outset, NarrowContent, Appearances } from 'components/content/content';
+import HeadingBlock from 'components/headingBlock/headingBlock';
+import Text, { LargeParagraph, SansParagraph } from 'components/text/text';
+
+const stories = storiesOf('Layout', module)
+  .addDecorator(withKnobs);
+
+const appearanceKnob = () => radios('Appearance', Object.keys(Appearances), Object.keys(Appearances)[0]);
+stories.add('Content', () => (
+  <div>
+    <Content
+      appearance={appearanceKnob()}
+    >
+      <Text title="Meet Content, your base grid">
+        <LargeParagraph>Wrap your content inside <code>Content</code> to have a grid basis</LargeParagraph>
+        <Divider />
+        <p>You can throw in multilines to break up pieces of text</p>
+      </Text>
+    </Content>
+    <Content
+      appearance="feature"
+    >
+      <Text>
+        <LargeParagraph>
+          Two pieces of content one after the other having the same appearance get a line between them
+        </LargeParagraph>
+        <SansParagraph>
+          All <code>Text</code> elements automatically get a max-width applied to them
+        </SansParagraph>
+      </Text>
+      <NarrowContent>
+        You can wrap custom non-text elements that need the same max-width as
+        the text in <code>NarrowContent</code>. It looks just like <code>Text</code> but
+        semantics matter! Don{'\''}t wrap non text elements in <code>Text</code> just to get a max width
+      </NarrowContent>
+    </Content>
+    <Content >
+      <Text title="Feeling adventurous ?" />
+      <Outset>
+        <Text>
+          {/* eslint-disable-next-line */}
```

Comment:
any reason why we don't want eslint here?

Reply:
it was complaining about the emoji and the fix was breaking flow and I couldn't run into any easy fixes so I figured since we shouldn't be using emojis in the actual pages anyway this was just not worth fixing further